templates/swagger/v1_json.tmpl

```diff
@@ -16284,7 +16291,7 @@
     "parameterBodies": {
       "description": "parameterBodies",
       "schema": {
-        "$ref": "#/definitions/MigrateRepoOptions"
+        "$ref": "#/definitions/PullReviewRequestOptions"
```

Comment:
I wonder why this line was changed ...

Reply:
i think because you have changed the **last** item to this in otions ... just igonre it